### ISSUE TITLE
cross platform cli api

### ISSUE THREAD

**Issue by thomasgeissl** [openframeworks/projectGenerator#243]
hello,
it would be very nice to have a cross platform cli api. (e.g. -o instead of \o on windows)
i could add cxxopts or any other option parser.
but i am not sure where it would break things. i guess the pg is currently used to create the platform packages, is not it?
i am happy to make this change if you are fine with it.
thomas

**Comment by arturoc** [comment]
yeah the options parser is not very good and it has other problems so having a better one would be useful i think

**Comment by thomasgeissl** [comment]
i had good experiences with cxxopts. i try to get it done this week.

**Comment by thomasgeissl** [comment]
https://github.com/thomasgeissl/ofProjectGenerator/tree/refactor/cliarguments

i still need to test and update the readme. but in case someone wantes to give feedback already, here it is.
